monai/data/csv_saver.py

```diff
@@ -24,7 +24,7 @@ class CSVSaver:
     the cached data into CSV file. If no meta data provided, use index from 0 to save data.
     """
 
-    def __init__(self, output_dir="./", filename="predictions.csv", overwrite=True):
+    def __init__(self, output_dir: str = "./", filename: str = "predictions.csv", overwrite: bool = True):
         """
         Args:
             output_dir (str): output CSV file directory.
```

Comment:
@benjamin-gorman @crtrentz You need to remove the types in the comments. 

Once you define `output_dir: str`  using PEP-484 you need to change `output_dir (str):` to `output_dir:`